cardano-coin-selection.cabal

```diff
@@ -13,7 +13,7 @@ license-file:        LICENSE
 category:            Web
 build-type:          Simple
 extra-source-files:  README.md
-cabal-version:       >=1.10
+cabal-version:       2.0
```

Comment:
Moving to a newer `cabal` version is necessary to enable the [internal libraries](https://www.haskell.org/cabal/users-guide/developing-packages.html?highlight=build%20tool%20depends#sublibs) feature.